tests/integration/ceph_base_file_test.go

```diff
@@ -178,16 +178,12 @@ spec:
     image: krallin/ubuntu-tini
     command:
         - "/usr/local/bin/tini"
+        - "-g"
         - "--"
         - "bash"
         - "-c"
         - |
-          pid=
-          trap 'exit 0' SIGTERM
-          trap '[[ $pid ]] && kill "$pid"' EXIT
-          sleep 1800 & pid=$!
-          wait $pid
-          pid=
+          sleep 1800
```

Comment:
Can we remove `bash` from the command? If we're using `tini` i would think this should work: `tini -- sleep 1800`